Assets/AppCenter/Plugins/AppCenterSDK/Auth/Android/AuthInternal.cs

```diff
@@ -31,6 +31,32 @@ public static void AddNativeType(List<IntPtr> nativeTypes)
             nativeTypes.Add(_auth.GetRawClass());
         }
 
+        public static AppCenterTask<UserInformation> SignInAsync()
+        {
+            var future = _auth.CallStatic<AndroidJavaObject>("signIn");
+            var javaTask = new AppCenterTask<AndroidJavaObject>(future);
+            var signInTask = new AppCenterTask<UserInformation>();
+            javaTask.ContinueWith(t =>
+            {
+                var exception = t.Result.Call<AndroidJavaObject>("getException");
+                if (exception != null)
+                {
+                    //TODO handle exception
+                }
+                var userInfo = t.Result.Call<AndroidJavaObject>("getUserInformation");
+                var aid = userInfo.Call<string>("getAccountId");
```

Comment:
@yuqche Let's use meaningful variable names, no abbreviations or short-handed names.

`aid` -> `accountId`
`atoken` -> `accessToken`
`it` -> `idToken`